packages/plugin-lighthouse/src/lib/constants.ts

```diff
@@ -63,3 +64,21 @@ async function loadLighthouseAudit(
   };
   return module.default;
 }
+
+export const DEFAULT_CLI_FLAGS: Partial<CliFlags> = {
+  // default values extracted from
+  // https://github.com/GoogleChrome/lighthouse/blob/7d80178c37a1b600ea8f092fc0b098029799a659/cli/cli-flags.js#L80
+  verbose: false,
+  quiet: false,
+  saveAssets: false,
+  // needed to pass CI on linux and windows (locally it works without headless too)
+  chromeFlags: '--headless=new',
```

Comment:
This should be `--headless=shell` 

https://github.com/puppeteer/puppeteer/blob/puppeteer-v22.1.0/docs/api/puppeteer.browserlaunchargumentoptions.md

This was checked by checking the version of lighthouse you have installed and the version of puppeteer they use. 

As users may use a different version of puppeteer it my be different for them. 

The may only have the option to have the boolean or the `new`.